### PR TITLE
Entities db

### PR DESCRIPTION
**Pull request by clemcx**
implementing distant database & new entities

### CODE REVIEW

**Review by RMayeux — APPROVED** [review]
approved

File: src/Entity/Advisor.php [hunk 1]
```diff
@@ -69,6 +69,17 @@ class Advisor
      */
     private $boards;
 
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $commentary;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Resume", inversedBy="advisor")
```
Comment (benj-san) [inline src/Entity/Advisor.php]: Here I see an issue with attribute naming, I believe there is a collection of advisors in the entity resume, that will be retrieved by the attribute advisor**s** . Please fix this little issue

File: src/Entity/Resume.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $content;
+
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Advisor", mappedBy="resume")
+     */
+    private $advisor;
```
Comment (benj-san) [inline src/Entity/Resume.php]: I specified why you should change this attribute above.

**Review by benj-san — COMMENTED** [review]
Fix the attribute plural name issue and I shall approve this P.R.

File: src/Entity/Advisor.php (hunk 1 above)
Comment (clemcx) [reply src/Entity/Advisor.php]: I made mistakes when creating the relation between Advisor and Resume. It should have been a OnetoMany from Advisor

**Review by benj-san — APPROVED** [review]

**Pull request merged**